### DISCUSSION TITLE
Does this work for Azure default domains, specifically, *.cloudapp.azure.com?

### DISCUSSION THREAD

**Discussion by SvenAelterman** [polymind-inc/acmebot#608] [Q&A] [answered]
Hi everyone,

I am new to this project. Based on what I see, Let's Encrypt domain validation is done using DNS. Based on that, the default Azure domains, like *.cloudapp.net for public IPs, wouldn't work.

Is there an alternative validation method available, such as URL-based?

Thanks.

**Comment by shibayan** [comment]
My understanding is that a certificate for a domain related to cloudapp.net cannot be obtained using any method.

https://community.letsencrypt.org/t/azure-support-cloudapp-net/8107/5

**Reply by SvenAelterman** [reply]
Thanks for the quick reply!

I am sorry, I used that as an example and it was a poor one. Here's a better one: *.cloudapp.azure.com. That one is possible: the AzureTRE project does it (of course not using your solution but using a custom script).

![image](https://github.com/shibayan/keyvault-acmebot/assets/17446043/e07cc7f6-9013-4029-b113-24326d055e28)

**Reply by shibayan** [reply, marked as answer]
I did not know that LE had issued a certificate to this domain.

Unfortunately, this project does not plan to support URL-based (HTTP-01) authentication, so you will need to use a custom domain.
